### PR TITLE
Add getUrl to DOWNLOAD_METHODS

### PR DESCRIPTION
**Pull request by djmitche**
One more thing I missed in the object service, when adding this new method :(

### CODE REVIEW

**Review by lotas — APPROVED** [review]
LGTM

File: services/object/test/helper/backend-general.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+const request = require('superagent');
```
[the remark this answers is not in the document]
Comment (djmitche) [reply services/object/test/helper/backend-general.js:1]: I will give credit to `goimports` for allowing me to forget to look for unused dependencies :)

**Comment by djmitche** [comment]
Looks like the "notify" service test flaked with a timeout..

File: services/object/test/helper/backend-general.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (petemoore) [reply services/object/test/helper/backend-general.js]: I think somebody has hijacked Dustin's account, the real Dustin would never praise the go ecosystem!

**Review by lotas — APPROVED** [review]

**Review by petemoore — APPROVED** [review]

**Pull request merged**